sky/core.py

```diff
@@ -64,6 +64,30 @@ def status(refresh: bool = False) -> List[Dict[str, Any]]:
     return cluster_records
 
 
+@usage_lib.entrypoint
+def report() -> List[Dict[str, Any]]:
+    # NOTE(dev): Keep the docstring consistent between the Python API and CLI.
+    """Get all cluster cost reports, including those that have been downed.
+
+        The following fields for each cluster are recorded: cluster name,
+        resources, launched time, duration that cluster was up,
+        and total cost.
+
+        The estimated cost column indicates price for the cluster based on the
+        type of resources being used and the duration of use up until the call
+        to status. This means if the cluster is UP, successive calls to report
+        will show increasing price. The estimated cost is calculated based on
+        the local cache of the cluster status, and may not be accurate for
+        the cluster with autostop/use_spot set or terminated/stopped
+        on the cloud console.
+
+        Returns:
+        A list of dicts, with each dict containing the cost information of a
+        cluster.
+    """
+    return global_user_state.get_clusters_from_history()
```

Comment:
We should include the total cost in the dict returned instead of calculating it in the `status_utils`. The `status_utils` should only contain the logic for formating the output string.